main.c

```diff
@@ -338,7 +411,7 @@ on_handler_appeared(GDBusConnection *connection,
 	if (info->register_invocation) {
 		info->connection = connection;
 		tcmur_register_dbus_handler(handler);
-		dbus_export_handler(handler, G_CALLBACK(on_dbus_check_config));
+		dbus_export_handler(handler, G_CALLBACK(on_dbus_check_config), G_CALLBACK(on_dbus_mediachange));
```

Comment:
Could you break this up to 80 cols here and below.

Reply:
Yes, will change